src/main/java/com/fasterxml/aalto/sax/SAXParserImpl.java

```diff
@@ -262,6 +263,8 @@ public boolean getFeature(String name)
             case IS_STANDALONE: // read-only, but only during parsing
                 // !!! TBI
                 return true;
+            case EXTERNAL_GENERAL_ENTITIES:
```

Comment:
Is this true? Are semantics of `EXTERNAL_GENERAL_ENTITIES` inverse of `RETAIN_ATTRIBUTE_GENERAL_ENTITIES`?

Reply:
That description would suggest it applies to ALL general entities. But what does "include" mean? Reported as specific event vs expanded (seems logical)?

Point is: feature being added here does not apply to all general entities but simply ones that are part of attribute values.

I realize that SAX property descriptions are vague and what matters is actual behavior, so maybe testing what Xerces does would be useful.

Reply:
Ok: I think this part should be removed; I can do that after merge it's fine. Conceptually it's just not the same feature and should not be reported as such. It is Aalto-specific settings beyond standard-specified features.

I'd be ok merging this PR, but just need CLA now.

Reply:
Got it! Will try to get this reviewed and added by end of the week.